--- debian/control.orig
+++ debian/control
@@ -45,8 +45,8 @@
                modemmanager-dev,
                pkgconf,
                python3-docutils,
-               systemd-dev
-Build-Conflicts: libfdk-aac-dev
+               systemd-dev,
+               libfdk-aac-dev
 Standards-Version: 4.7.0
 Vcs-Browser: https://salsa.debian.org/utopia-team/pipewire
 Vcs-Git: https://salsa.debian.org/utopia-team/pipewire.git
--- debian/rules.orig
+++ debian/rules
@@ -67,7 +67,7 @@
                -Dauto_features=enabled \
                -Davahi=enabled \
                -Dbluez5-backend-native-mm=enabled \
-               -Dbluez5-codec-aac=disabled \
+               -Dbluez5-codec-aac=enabled \
                -Dbluez5-codec-aptx=enabled \
                -Dbluez5-codec-lc3=enabled \
                -Dbluez5-codec-lc3plus=disabled \
